modules/apps/shared-dependencies/shared-dependencies-apache-http-components/bnd.bnd

```diff
@@ -1,30 +1,4 @@
 Bundle-Name: Liferay Shared Dependencies Apache Http Components
 Bundle-SymbolicName: com.liferay.shared.dependencies.apache.http.components
 Bundle-Version: 1.0.2
--exportcontents:\
-	org.apache.http,\
-	org.apache.http.auth,\
-	org.apache.http.client,\
-	org.apache.http.client.config,\
-	org.apache.http.client.entity,\
-	org.apache.http.client.methods,\
-	org.apache.http.client.protocol,\
-	org.apache.http.client.utils,\
-	org.apache.http.concurrent,\
-	org.apache.http.config,\
-	org.apache.http.conn,\
-	org.apache.http.conn.routing,\
-	org.apache.http.conn.socket,\
-	org.apache.http.conn.ssl,\
-	org.apache.http.cookie,\
-	org.apache.http.entity,\
-	org.apache.http.entity.mime,\
-	org.apache.http.entity.mime.content,\
-	org.apache.http.impl.client,\
-	org.apache.http.impl.conn,\
-	org.apache.http.impl.cookie,\
-	org.apache.http.message,\
-	org.apache.http.params,\
-	org.apache.http.pool,\
-	org.apache.http.protocol,\
-	org.apache.http.util
\ No newline at end of file
+-exportcontents: org.apache.http.*
```

Comment:
@jeffwu0724 Please list all the packages you want to export rather than using *, it is hard to track what are exported what are not.

Tina